test/test_helpers/simple_mini_test.rb

```diff
@@ -1,8 +1,3 @@
 # frozen_string_literal: true
 
-class SimpleMiniTest < ActiveSupport::TestCase
-  def teardown
-    super
-    User.current = nil
-  end
-end
+class SimpleMiniTest < ActiveSupport::TestCase; end
```

Comment:
Why keep this class anymore?

Reply:
Multiple tests inherit from it, didn't want to clutter the PR even more than it already is. But can easily be removed.